data/mods/blazemod/blaze_blob.json

```diff
@@ -1,1238 +1,1660 @@
+    "color": "green",
+    "name": "growing blob frame",
+    "description": "A growing vehicle frame made of bone.  It's coated in a thick layer of ooze, though there's greater amounts on the bindings.  It's not quite tough enough to use.",
     "price": 100,
-    "material": ["steel", "clay"],
-    "weight": 17100,
-    "volume": 50,
+    "material": ["bone", "flesh"],
+    "weight": 3662,
+    "volume": 40,
     "bashing": 0,
+    "cutting": 0,
     "to_hit": -2,
+    "max_charges": 0,
+    "initial_charges": 0,
+    "charges_per_use": 0,
+    "turns_per_charge": 0,
+    "revert_to": "null",
     "category" : "tools",
     "use_action": {
         "type": "delayed_transform",
-        "transform_age": 7200,
-        "not_ready_msg": "While it definitely smells ripe, it isn't done.",
-        "msg": "You hold your breath and open up the container.",
-        "moves": 50,
-        "container": "blob_tank",
-        "target": "bfeed",
-        "target_charges": 5000
+        "transform_age": 36000,
+        "not_ready_msg": "You test the frame; it gives easily, it's still not sturdy enough",
+        "msg": "You test the frame; It wiggles, and then collapses into an lump of goo.",
+        "moves": 0,
+        "target": "gloople",
+        "target_charges": 0
     }
   },
   {
-    "id": "blob_tank",
-    "type": "CONTAINER",
-    "symbol": ")",
-    "color": "brown",
-    "name": "blob core",
-    "description": "A large container previously filled with some horrendously smelling liquid and now infested with an equally horrendously smelling unknown substance that seems to have permeated into the it.  Occasionally the container gives a shake, as if alive.",
-    "price": 100,
-    "material": ["flesh", "clay"],
-    "weight": 24100,
-    "volume": 50,
-    "bashing": 0,
-    "to_hit": -2,
-    "contains": 10,
-    "ammo": "bfeed",
-    "watertight": true,
-    "seals": true
-    },
-  {
-    "id": "blobshootergrow",
+    "id": "glooplegrow",
     "type": "TOOL",
     "symbol": ";",
-    "color": "brown",
-    "name": "growing blob shooter",
-    "description": "This appears to be some form of structure constructed from bone, held together by some unknown slimy substance.  A jury-rigged slow-drip dispenser trickles some fluid you'd prefer not to think too hard about throughout the structure.",
+    "color": "light_green",
+    "name": "growing gelatinous mass",
+    "name_plural" : "growing gelatinous mass",
+    "description": "Not quite fully grown, this blob requires nourishment to fully develop.",
     "price": 100,
-    "material": ["bone", "flesh"],
-    "weight": 13115,
-    "volume": 14,
+    "material": ["flesh"],
+    "weight": 3562,
+    "volume": 8,
     "bashing": 0,
+    "cutting": 0,
     "to_hit": -2,
+    "max_charges": 0,
+    "initial_charges": 0,
+    "charges_per_use": 0,
+    "turns_per_charge": 0,
+    "revert_to": "null",
```

Comment:
All of those are redundant:
* "max_charges": 0,
* "initial_charges": 0,
* "charges_per_use": 0,
* "turns_per_charge": 0,
* "revert_to": "null",